src/test/scala/org/broadinstitute/dsde/rawls/graph/OrientDbTestFixture.scala

```diff
@@ -0,0 +1,27 @@
+package org.broadinstitute.dsde.rawls.graph
+
+import java.util.logging.{Logger, LogManager}
+
+import com.tinkerpop.blueprints.impls.orient.{OrientBaseGraph, OrientGraph}
+import org.scalatest.{FunSuite, BeforeAndAfter}
+
+trait OrientDbTestFixture extends FunSuite with BeforeAndAfter {
```

Comment:
does this trait have to extend FunSuite? what if I want my test to use FlatSpec?